commands/cloudapi-v6/ipblock.go

```diff
@@ -233,20 +241,72 @@ func RunIpBlockUpdate(c *core.CommandConfig) error {
 }
 
 func RunIpBlockDelete(c *core.CommandConfig) error {
-	if err := utils.AskForConfirm(c.Stdin, c.Printer, "delete ipblock"); err != nil {
-		return err
-	}
-	c.Printer.Verbose("Ip block with id: %v is deleting...", viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgIpBlockId)))
-	resp, err := c.CloudApiV6Services.IpBlocks().Delete(viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgIpBlockId)))
-	if resp != nil {
-		c.Printer.Verbose(cloudapiv6.RequestTimeMessage, resp.RequestTime)
+	var resp *resources.Response
+	allFlag := viper.GetBool(core.GetFlagName(c.NS, cloudapiv6.ArgAll))
+	ipBlockId := viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgIpBlockId))
+	if allFlag {
+		err := DeleteAllIpBlocks(c)
+		if err != nil {
+			return err
+		}
+	} else {
+		if err := utils.AskForConfirm(c.Stdin, c.Printer, "delete ipblock"); err != nil {
+			return err
+		}
+		c.Printer.Verbose("Starting deleting Ip block with ID: %v...", ipBlockId)
+		resp, err := c.CloudApiV6Services.IpBlocks().Delete(ipBlockId)
```

Comment:
Use = instead of := to not define resp as a new variable. It will not be recognized at the end of the current function.